lua/wikis/commons/ParticipantTable/Base.lua

```diff
@@ -290,9 +289,12 @@ function ParticipantTable:store()
 	local placements = self:getPlacements()
 
 	Array.forEach(self.sections, function(section) Array.forEach(section.entries, function(entry)
-		local lpdbData = Opponent.toLpdbStruct(entry.opponent)
+		local opponent = entry.opponent
+		-- do not store team opponents
+		if opponent.type == Opponent.team then return end
+		local lpdbData = Opponent.toLpdbStruct(opponent)
 
-		if section.config.noStorage or Opponent.isTbd(entry.opponent) or Opponent.isEmpty(entry.opponent) then return end
+		if section.config.noStorage or Opponent.isTbd(opponent) or Opponent.isEmpty(opponent) then return end
```

Comment:
IMO move this to the top and include the team type check

Reply:
to the top meaning where?
if we were to have a part table that has both team and non team opponents wouldn't we want to still store the non team opponents? (hence why i did the check here)

Reply:
294?